client/stylesheets/_toCat.scss

```diff
@@ -3285,6 +3281,10 @@ On click the images will be displayed at normal size to complete the effect
     font-size: 0.9rem;
 }
 
+.profileUserRating ul li span a {
+    color: #404040 !important;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces